src/style.css

```diff
@@ -1,3 +1,85 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-weight: 450;
+  font-family: Roboto, sans-serif;
+  font-size: 15px;
+  line-height: 14.06px;
```

Comment:
The line height is set to a non-standard value of 14.06px. It's recommended to use a unitless value or a value that is a multiple of the font size for better readability and consistency.